src/Niekert/DiscordMCPE/Main.php

```diff
@@ -25,7 +25,7 @@ public function onEnable()
             return;
         }
         if ($this->pp) {
-            if (is_null(this->getServer()->getPluginManager()->getPlugin('PurePerms'))) {
+            if (is_null($this->getServer()->getPluginManager()->getPlugin('PurePerms'))) {
```

Comment:
supposed to be checking if it isnt null, not if it is null oops lul

Reply:
you forgot !

Reply:
that's why i said that.